### PR TITLE
docs: add thumbnails for more packages

### PR DESCRIPTION
**Pull request by chris-at-jobber**
## Motivations

Getting ahead of us adding docs for the ThemeContext and our linter packages

### Added

- More thumbnails!

### Removed

- Duplicated SVG file

## Testing

Try using the thumbnails? Can also just validate that they exist

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

File: packages/site/public/Comp/distress_paper-rip.svg
Comment (chris-at-jobber) [inline packages/site/public/Comp/distress_paper-rip.svg:1]: This file was a duplicate

File: packages/site/public/ESLintConfig.png
Comment (chris-at-jobber) [inline packages/site/public/ESLintConfig.png:1]: If you have a better idea for this thumbnail, I'm open to it

File: packages/site/public/AtlantisThemeContext.png
Comment (chris-at-jobber) [inline packages/site/public/AtlantisThemeContext.png:1]: I think this gets the idea across, it's a "dev-centric" component but we can hint at the outcomes of it in a minimal way

File: packages/site/public/Hooks.png
Comment (chris-at-jobber) [inline packages/site/public/Hooks.png:1]: Get it? It's a React Hook!

File: packages/site/public/Comp/distress_paper-rip.svg
Comment (chris-at-jobber) [reply packages/site/public/Comp/distress_paper-rip.svg:1]: (can validate by checking the collection views)
image

**Review by scotttjob — APPROVED** [review]

**Pull request merged**